tests/test_taar_amowhitelist.py

```diff
@@ -92,7 +92,7 @@
             "text_count": 1.0
         },
         "summary": {
-            "en-US": "Luffi - Let\u00b4s use freeware! fun included:)"
+            "en-US": "Luffi - Let\\u00b4s use freeware! fun included:)"
```

Comment:
This unicode change does get added in 2to3, and I end up reversing this.

Reply:
to be clear, using a `\\` correctly does the same thing in python 2 and 3, where reducing it to `\u` behaves differently (unless it used `u"`, then it behaves the same for both)

Reply:
I'm not totally clear on this behavior, but I've left it as is.